nifi-toolkit/pom.xml

```diff
@@ -23,7 +23,6 @@
     <artifactId>nifi-toolkit</artifactId>
     <packaging>pom</packaging>
     <modules>
-        <module>nifi-toolkit-tls</module>
```

Comment:
The `tls-toolkit` is also referenced by the `nifi-toolkit/nifi-toolkit-assembly`'s `docker` build and tests, it will need removing

Reply:
Thanks!  Missed those...